src/nodes/array/unshift.js

```diff
@@ -0,0 +1,61 @@
+const dayjs = require('dayjs')
+const AbstractNode = require('@bluepjs/vm/src/nodes/abstract')
+
+class ArrayUnshift extends AbstractNode {
+
+  static metadata() {
+    return {
+      name: 'Unshift',
+      code: 'array/unshift',
+      type: 'modifier',
```

Comment:
shift, pop, push, unshift are not modifiers.
they change values of inputs, so they should be callables and has execution flow input/output